### PR TITLE
Update doc.raw() to return the entire raw JSON

### PR DESCRIPTION
**Pull request by x389liu**

### CODE REVIEW

File: src/main/java/io/anserini/collection/JsonCollection.java [hunk 1]
```diff
@@ -164,10 +166,10 @@ public String contents() {
 
     @Override
     public String raw() {
-      if (contents == null) {
-        throw new RuntimeException("JSON document has no \"contents\" field");
+      if (raw == null) {
+        throw new RuntimeException("JSON document has no \"raw\" field");
```
Comment (lintool) [inline src/main/java/io/anserini/collection/JsonCollection.java:170]: I don't think `raw` can ever be `null`? Can you check? If `raw` is `null`, then how did we read the document in the first place...?

**Review by lintool — COMMENTED** [review]
See comment?

File: src/main/java/io/anserini/collection/JsonCollection.java (hunk 1 above)
Comment (lintool) [reply src/main/java/io/anserini/collection/JsonCollection.java:170]: And either way, I don't think this is the right error message... "raw" isn't a field in the JSON...

Comment (x389liu) [reply src/main/java/io/anserini/collection/JsonCollection.java]: Thanks for pointing this out, `raw` can never be `null` as indexed docs always have `id`/`contents`.

**Review by lintool — APPROVED** [review]

**Pull request merged**